packages/react-tabster/tsconfig.json

```
@@ -1,17 +1,17 @@
 {
   "extends": "../../tsconfig.base.json",
+  "include": ["src"],
   "compilerOptions": {
     "target": "ES5",
-    "lib": ["ES2015", "DOM"],
+    "module": "CommonJS",
+    "lib": ["es5", "dom"],
```

Same issue as #18682

```suggestion
    "lib": ["ES2015", "dom"],
```